documentation/src/theme/Footer/Layout/index.js

```diff
@@ -178,19 +178,19 @@ export default function FooterLayout({
                                     rel="noreferrer"
                                 >
                                     {socialLink.label === "github" && (
-                                        <GithubIcon className="w-[22px] h-[22px] hover:fill-[#171515]" />
+                                        <GithubIcon className="w-[22px] h-[22px] hover:fill-[#171515] hover:fill-[#171515] transition-colors ease-in-out duration-500" />
                                     )}
                                     {socialLink.label === "twitter" && (
-                                        <TwitterIcon className="w-[22px] h-[22px] hover:fill-[#1DA1F2]" />
+                                        <TwitterIcon className="w-[22px] h-[22px] hover:fill-[#1DA1F2] hover:fill-[#171515] transition-colors ease-in-out duration-500" />
                                     )}
                                     {socialLink.label === "linkedin" && (
-                                        <LinkedinIcon className="w-[22px] h-[22px] hover:fill-[#0077B5]" />
+                                        <LinkedinIcon className="w-[22px] h-[22px] hover:fill-[#0077B5] hover:fill-[#171515] transition-colors ease-in-out duration-500" />
```

Comment:
```suggestion
                                        <LinkedinIcon className="w-[22px] h-[22px] hover:fill-[#0077B5]  transition-colors ease-in-out duration-500" />
```
Do we need this ?
It doesn't cause any problems on hover effect, but it is confusing

Reply:
hover effect on icon isn't smooth that's why I made it smooth

Reply:
Transitions are okay, Thank you. 🙏 

But icons have 2 hover class right now and one is ovveriding another. This is confusing.
We should use 1 hover class on html elements.

Reply:
ok I'm doing it now